packages/mantine/src/components/modal-wizard/ModalWizard.tsx

```diff
@@ -120,7 +137,7 @@ export const ModalWizard: ModalWizardType = ({
     return (
         <Modal
             opened={opened}
-            classNames={classNames}
+            classNames={{modal: cx(modal, classNames?.modal), body: cx(body, classNames?.body)}}
```

Comment:
I think you can do something like this if you follow the guide properly https://mantine.dev/guides/custom-components/

```suggestion
            className={cx(modal, body, className)}
```